tsl/test/sql/backup.sql

```diff
@@ -0,0 +1,106 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+
+\unset ECHO
+\o /dev/null
+\ir include/remote_exec.sql
+\ir include/filter_exec.sql
+\o
+\set ECHO all
+
+\set MY_DB1 :TEST_DBNAME _1
+\set MY_DB2 :TEST_DBNAME _2
+\set MY_DB3 :TEST_DBNAME _3
+
+SELECT * FROM add_data_node('data_node_1', host => 'localhost', database => :'MY_DB1');
+SELECT * FROM add_data_node('data_node_2', host => 'localhost', database => :'MY_DB2');
+SELECT * FROM add_data_node('data_node_3', host => 'localhost', database => :'MY_DB3');
+GRANT USAGE ON FOREIGN SERVER data_node_1, data_node_2, data_node_3 TO PUBLIC;
+
+-- Import testsupport.sql file to data nodes
+\unset ECHO
+\o /dev/null
+\c :MY_DB1
+SET client_min_messages TO ERROR;
+\ir :TEST_SUPPORT_FILE
+\c :MY_DB2
+SET client_min_messages TO ERROR;
+\ir :TEST_SUPPORT_FILE
+\c :MY_DB3
+SET client_min_messages TO ERROR;
+\ir :TEST_SUPPORT_FILE
+\c :TEST_DBNAME :ROLE_SUPERUSER;
+\o
+SET client_min_messages TO NOTICE;
+\set ECHO all
+
+-- test NULL (STRICT function)
+SELECT create_distributed_restore_point(NULL);
+
+-- test long restore point name (>= 64 chars)
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('0123456789012345678901234567890123456789012345678901234567890123456789');
+\set ON_ERROR_STOP 1
+
+-- test super user check
+\set ON_ERROR_STOP 0
+SET ROLE :ROLE_1;
+SELECT create_distributed_restore_point('test');
+RESET ROLE;
+\set ON_ERROR_STOP 1
+
+-- make sure 2pc are enabled
+SET timescaledb.enable_2pc = false;
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('test');
+\set ON_ERROR_STOP 1
```

Comment:
I don't follow how 2PC is enabled back. Is it reset automatically after one transaction? Can you do `SHOW timescaledb.enable_2pc`, so it is clear that it is enabled back?

Reply:
Good catch, likely accidentally removed it while refactoring

Reply:
Then why have the regression test succeeded?

Reply:
Because we reconnected in tests after that and it set it to the default value

Reply:
Thank you for clarifying! Basically it became different session.